test/models/api_user_test.rb

```diff
@@ -47,11 +47,20 @@ def test_access_to_returns_false_if_user_has_no_attribute_access
     refute api_user.access_to?(ProtectedArea, :designation)
   end
 
+  def test_access_to_returns_true_if_user_has_attribute_access
+    api_user = create(:api_user, permissions: {"ProtectedArea" => ["name", "is_green_list"]})
+    assert api_user.access_to?(ProtectedArea, :is_green_list)
+  end
+
+  def test_access_to_returns_false_if_user_has_no_attribute_access
+    api_user = create(:api_user, permissions: {"ProtectedArea" => ["name", "is_green_list"]})
+    refute api_user.access_to?(ProtectedArea, :designation)
+  end
+
```

Comment:
So these two tests are intrinsically testing the same behaviour of the previous ones; it's just the attribute that changes, but the behaviour is the same. I guess it's fine keeping these tests so we are absolutely sure we haven't misspelled anything.
The only thing you need to make sure is that you use a different name for the test functions; if the name is the same, ruby will pick only one of the existing tests with the same name.

Reply:
Thanks @Levia I have renamed these tests.

Reply:
Also, since that very same behaviour has already been tested, it would be good to make different ones, such has access/no_access to specific attribute.
For example assert user has access, so if the api_user has access to `is_green_list` then assert that he can actually access it, or refute user access to the same attribute; in this case, when you create a user, make sure that you are giving him access to another resource, and then test if he has access to `is_green_list`